src/js/controllers/mapController.ts

```diff
@@ -434,6 +435,13 @@ export class MapController {
     }
   }
 
+  attachCoordinatesWidget(domref: React.MutableRefObject<any>): void {
+    const ccWidget = new CoordinateConversion({
```

Comment:
```typescript
  attachCoordinatesWidget(domref: React.MutableRefObject<any>): void {
    new CoordinateConversion({
      view: this._mapview,
      container: domref.current
    });
  }

```
I'm thinking we won't need to set a new instance of `CoordinateConversion` to a constant, since it won't be accessed in other parts of `mapController.ts` or the app?